javascript/ql/src/semmle/javascript/frameworks/React.qll

```diff
@@ -52,10 +52,10 @@ abstract class ReactComponent extends ASTNode {
   }
 
   /**
-   * Gets a `this` access in an instance method of this component.
+   * Gets the `this` node in an instance method of this component.
    */
   DataFlow::SourceNode getAThisAccess() {
```

Comment:
The name of this predicate no longer matches its description. Should we deprecate and rename it?

Reply:
Yes, let's do that.

Reply:
I've just renamed it to `getAThisNode`. It seems a more high-level concept would be more useful, like getting a reference to an instance of the component, although that's sort of what `ref()` is doing, so I've just done the simplest possible renaming.